public/database.php

```diff
@@ -66,6 +66,19 @@ function select_posts_preview(string $board = NULL, int $parent = 0, int $offset
   return $sth->fetchAll();
 }
 
+function count_replies(string $board = NULL, int $parent) : int {
```

Comment:
Shouldn't default value arguments be the last ones? I think you can just remove the = NULl default value.